homeassistant/components/isy994/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Universal Devices ISY/IoX",
   "integration_type": "hub",
   "documentation": "https://www.home-assistant.io/integrations/isy994",
-  "requirements": ["pyisy==3.1.3"],
+  "requirements": ["pyisy==3.1.4"],
```

Comment:
The version bump doesn't seem related to the deprecation. I recommend splitting the PR in two.

Reply:
Done.